src/locale/translations/region.json

```diff
@@ -182,6 +207,12 @@
             "Body": "Quand vous recevez un résultat de test positif pour la COVID-19, la Santé publique vous fournit aussi une clé à usage unique.\n\nVous recevrez une seule clé, et elle expirera au bout de 24 heures.",
             "CTA": "",
             "Link": ""
+          },
+          "SK": {
+            "Title": "Obtenir une clé à usage unique",
+            "Body": "Quand vous recevez un résultat de test positif pour la COVID-19, la Santé publique vous fournit aussi une clé à usage unique.\n\nVous recevrez une seule clé, et elle expirera au bout de 24 heures.\n\nSi vous avez un problème avec votre clé, contactez l’autorité sanitaire qui vous a donné votre résultat de test.",
+            "CTA": "",
+            "Link": ""
```

Comment:
Sorry, I missed that the link is missing here: www.saskatchewan.ca/residents/health/public-health/public-health-offices